t/03-testapi.t

```diff
@@ -1047,16 +1049,34 @@ like(exception { ensure_installed }, qr/implement.*for your distri/, 'ensure_ins
 lives_ok { hold_key('ret') } 'hold_key can be called';
 lives_ok { release_key('ret') } 'release_key can be called';
 lives_ok { reset_consoles } 'reset_consoles can be called';
+
+subtest 'assert/check recorded sound' => sub {
+    $cmds = [];
+    lives_ok { start_audiocapture } 'start_audiocapture can be called';
+    like $cmds->[0]->{filename}, qr/captured\.wav/, 'audiocapture started with expected args' or diag explain $cmds;
+    my $mock_testapi = Test::MockModule->new('testapi');
+    $mock_testapi->noop('_snd2png');
+    $mock_basetest->noop('verify_sound_image');
+    ok assert_recorded_sound('foo'), 'assert_recorded_sound can be called';
+    ok check_recorded_sound('foo'), 'check_recorded_sound can be called';
```

Comment:
It would be nice to test whether these functions do what they're expected to do.

Reply:
yes, it would be nice but as I explained in https://github.com/os-autoinst/os-autoinst/pull/1696#issuecomment-1410316512 so far I only made sure the functions can be called as-is regardless of what they actually do. There is still work to be done outside this PR to have full test coverage of testapi.pm but this PR is only about using signatures everywhere.